### ISSUE TITLE
Bot uses wrong pronouns [Bug]

### ISSUE THREAD

**Issue by PaleSovereign** [ZeroTwo-Bot/ZeroTwo-Bot#125]
Despite setting one's gender to female (and despite it appearing correctly when using zt!profile) the bot will still say "his" when using zt!daily for example.

**Comment by xAngel04** [comment]
Yes, the bot developer is aware of this. We have gotten this reported in Support, multiple times. Thank you for the report though.
